docs/service-mesh/14-03-connection-refused.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Connection refused" errors after upgrade
```

Comment:
We can't use quotation marks is metadata. The title needs to be rephrased and the exact error message should go into the doc content.

Reply:
Similarly to "installation" troubleshooting, I'd like to keep the error message in the document title so that's it's easier for the user to find even by looking at the list of available documents ;) I'll remove the q. marks.